### PR TITLE
 Added getImageReverted method in ABITrace.java 

### PR DESCRIPTION
**Pull request by el1jaC**
- getImageReverted method returns mirror image of sequence, caused by Reverse Complement alignment.

- General code refactoring (final variables initialization moved in constructor, fix class code identation)

### CODE REVIEW

**Comment by el1jaC** [comment]
It is the first pull request in my life, but i think the code of ABITrace is right.
Why Travis CI catches that error?

File: sequencing/src/main/java/org/biojava/bio/program/abi/ABITrace.java [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.
+ .
```
Comment (heuermh) [inline sequencing/src/main/java/org/biojava/bio/program/abi/ABITrace.java]: Not sure where these two new lines came from, they are failing the Travis build.

File: sequencing/src/main/java/org/biojava/bio/program/abi/ABITrace.java [hunk 2]
```diff
@@ -58,365 +54,500 @@
...
+    //the next three lines are the important persistent data
+    private String sequence;
+    private int A[], G[], C[], T[], Basecalls[], Qcalls[];
+    private int TraceLength, SeqLength;
+
+    //This is the actual file data.
+    private byte[] TraceData;
+
+    private final int maximum;
+    //the next four declaration lines comprise the file index information
```
Comment (heuermh) [inline sequencing/src/main/java/org/biojava/bio/program/abi/ABITrace.java]: The formatting in this comment, while strange before, is worse now.

**Comment by heuermh** [comment]
Thanks for the contribution, @el1jaC!

I would suggest rebasing to only one commit addressing "getImageReverted method returns mirror image of sequence, caused by Reverse Complement alignment" and leave out the refactoring and reformatting.

**Comment by el1jaC** [comment]
Fixed code based to your suggestions, Travis raise this error:

> /home/travis/build/biojava/biojava-legacy/gui/src/main/java/org/biojava/bio/gui/sequence/AbiTraceRenderer.java:[80,47] incompatible types: org.biojava.bio.symbol.AtomicSymbol cannot be converted to char

I edited the code directly from GitHub editor for this time, to check fixes.

**Comment by el1jaC** [comment]
Please close this issue-
I deleted my repo, and I'll recreate it with only one commit.

There was too much chaos in previous one.

Sorry :)

**Pull request closed without merging**